specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-03-31/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2019-12-31",
```

Comment:
should be changed to right api-version 2020-03-31? also for other example files

Reply:
Fixed